### DISCUSSION TITLE
Dryer WrinkleGuard

### DISCUSSION THREAD

**Discussion by marcolega** [ekutner/home-connect-hass#139] [General]
Hi, my dryear Bosch WTY877H8IT has a setting for WrinkleGuard but I can't see the sector in the integration, is it possible to add it?

Moreover using official app I can see an option selector wer I can set some program option like laundryCare, but I can't see any selector for the option in the integration, is it possible to add it also? 
I also have a BOSCH washing machine WAX32EH0II that has the same proble with the options

Let me know
Thanks
Marco.

**Comment by ekutner** [comment]
Read the FAQ section in the README
